lib/models/device.coffee

```diff
@@ -117,9 +117,6 @@ getDeviceModel = (deps, opts) ->
 		return url.resolve(dashboardUrl, "/apps/#{options.appId}/devices/#{options.deviceId}/summary")
 
 	addExtraInfo = (device) ->
-		# TODO: Move this to the server
-		device.application_name = device.application[0].app_name
-		device.dashboard_url = getDashboardUrl({ appId: device.application[0].id, deviceId: device.id })
```

Comment:
Strictly speaking the ui doesn't give a damn about the app id in dashboard, and using the wrong app id will still get you the right device (something that I abuse fairly often when I know the device id but not app id) - with that in mind it might be possible to give a url direct to a device that the dashboard can understand, cc @LucianBuzzo

Reply:
True. I don't think that matters much for this though - for now it's only used in the CLI, which is going to want to expand the app anyway, so I'll just extract the same logic unchanged to there and keep things simple. In a multi-app future, we'll definitely want device only URLs, but for now I'm not too worried.

Reply:
The UI will also need a getDashboardUrl method for resin-io/resin-ui#1351 .
Would it make sense to change that method to the suggested url and make this PR depend on the UI ticket?

Reply:
I imagine the UI wouldn't want a URL method anyway, since it can use internal routing mechanisms instead (tbc, no idea what our routing will look like in React land). And to be clear, the CLI definitely will keep this functionality, it's just where the logic lives that'll be changing!